### PR TITLE
Add LTO to compilations where it doesn't fail

### PR DESCRIPTION
**Pull request by diddledani**
Signed-off-by: Dani Llewellyn <diddledani@ubuntu.com>

### CODE REVIEW

**Comment by diddledani** [comment]
I must fix the build check - the check itself is broken right now so ignore the failure to build

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -246,13 +245,33 @@ parts:
...
     override-pull: |
       snapcraftctl pull
       sed -i 's/not stable, Description:/not stable, description:/' meson.build
     override-build: |
-      snapcraftctl build
+      /usr/bin/python3 -m pip install -U meson
+
+      # LTO fails on armhf
+      LTO=-Db_lto=true
+      if [ $SNAPCRAFT_ARCH_TRIPLET = "arm-linux-gnueabihf" ]; then
```
Comment (zyga) [inline snap/snapcraft.yaml:263]: Shell nitpick, you should quote the variable, quoting the word on the right doesn't help.

Comment (diddledani) [reply snap/snapcraft.yaml:263]: oops, you're right.. my bad

**Review by kenvandine — APPROVED** [review]
Nice improvement

**Pull request merged**